doc/en/fixture.rst

```diff
@@ -1136,8 +1136,8 @@ and teared down after every test that used it.
 
 .. _`usefixtures`:
 
-Using fixtures from classes, modules or projects
-----------------------------------------------------------------------
+Use fixtures implicitly with @pytest.mark.usefixtures
```

Comment:
I kinda prefer the old title here:

- The old one more directly states the main use case
- Not sure about the word "implicitly", I wouldn't say it's exactly implicit, and also has some overlap with autouse.

I think it'd be good to add the "with usefixtures" part though. However I'd omit the `@pytest.mark` part because this section also discusses the `usefixtures` ini option.

Reply:
Hmmm the old title read ambiguously to me, it could also meant "Using fixtures *defined* in other classes..."

But I agree with your points of course.

How about "Use fixtures in classes and modules with usefixtures"?

Reply:
:+1:

Reply:
Done, thanks